src/pages/HowItWorks.tsx

```diff
@@ -40,30 +45,38 @@ export default function HowItWorks() {
             learnMoreButtonLabel={buttons?.[1]}
           />
 
-          {/* 			<p className="tex		t-gray-300">Section - 8</p> */}
-          <Section8
-            TitleLabel={sectiontCovidTestExplanation?.title}
-            TitleSize="h1"
-            ContainerTitleProps="text-center ml-[5vw] mr-[5vw]"
-            //Image
-            sizeImg="pr90"
-            border="none"
-            route1={howItWorksStep1}
-            route2={howItWorksStep2}
-            route3={howItWorksStep3}
-            alt="Section 7"
-            //TitleText
-            TitleSize1="h3"
-            TitleLabel1={sectiontCovidTestExplanation?.titleLabelRecording}
-            TitleLabel2={sectiontCovidTestExplanation?.titleLabelProcessing}
-            TitleLabel3={sectiontCovidTestExplanation?.titleLabelResults}
-            TextSize="normal"
-            TextLabel1={sectiontCovidTestExplanation?.textLabelRecording}
-            TextLabel2={sectiontCovidTestExplanation?.textLabelProcessing}
-            TextLabel3={sectiontCovidTestExplanation?.textLabelResults}
-            ContainerTextProps=""
-            buttonLabel={sectiontCovidTestExplanation.button}
-          />
+          <div className='w-full mt-60 z-10'>
+
+          {/* Background img 2  */}
+          <Image src={howItWorksStep3} alt='background' className="absolute left-0 w-full object-cover z-0 h-[87rem]
+          xs:h-100 sm:h-[950px] md:h-[940px] lg:h-[100] xl:h-[890px]"/>
```

Comment:
`h-full` can be used to set `height: 100%`, instead of `h-100` and `h-[100]` I believe